crates/core/tedge_api/src/entity_store.rs

```diff
@@ -30,14 +30,6 @@ use std::fmt::Display;
 use std::path::Path;
 use thiserror::Error;
 
-/// Represents an "Entity topic identifier" portion of the MQTT topic
-///
-/// Example:
-/// - topic: `te/device/dev1/service/myservice/m//my_measurement`
-/// - entity id: `device/dev1/service/myservice`
-///
-/// <https://thin-edge.github.io/thin-edge.io/next/references/mqtt-api/#group-identifier>
-
```

Comment:
Might be good to attach this definition to a type or a module, rather than remove it.

Reply:
This comment is for `EntityTopicId`. The struct was moved to another module along with the comment, but the comment in this module wasn't deleted.

https://github.com/thin-edge/thin-edge.io/blob/0748986c88f16a14d9b06456a7157836c3d036c7/crates/core/tedge_api/src/mqtt_topics.rs#L275-L292